src/Conf.ml

```diff
@@ -155,6 +155,37 @@ let sparse =
   mk ~default Arg.(value & flag & info ["sparse"] ~doc ~env)
 
 
+let escape_chars =
+  let doc =
+    "How to escape chars. Can be set in a config file with a `escape-chars {hexadecimal,octal,minimal}` line."
+  in
+  let env = Arg.env_var "OCAMLFORMAT_ESCAPE_CHARS" in
+  let default = `Hexadecimal in
```

Comment:
I would guess that minimal would be the best default, thoughts?

Reply:
I prefer `` `Minimal` too :+1: